Sprintfinity3902/Entities/Items/SingleExplosionCloud.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Sprintfinity3902.Interfaces;
+using Sprintfinity3902.SpriteFactories;
+
+namespace Sprintfinity3902.Entities.Items
+{
+    public class SingleExplosionCloud : AbstractEntity
```

Comment:
What is the class used for? I didn't see a reference to it anywhere, but I likely just missed it.

Reply:
This is for the player to just display one "cloud". I believe the bombexplosionitem is most similar but a 7-"cloud" in a circle and not a lone "cloud". I will rename it to CloudItem to clear confusion and follow naming convention